### DISCUSSION TITLE
A crash after calling `Actor.Invoke()` using async/await

### DISCUSSION THREAD

**Discussion by CrazyguyFPV** [nxrighthere/UnrealCLR#247] [Help] [answered]
Using an event that's called via the suggested Actor.Invoke() works, but as soon as I continue the execution past it in any meaningful way, I get an unreal crash that response with:

LogOutputDevice: Warning; Script Stack (4 frames):
MainUI_C.ExecuteUbergraph_MainUI
MainUI_C.CreateNewAccountSuccess
MauiController_C.ExecuteUbergraph_MauiController
MauiController_C.CreateNewAccountReady
LogWindows: Windows GetLastError: The operation completed successfully. (0)

And then a complete Unreal Editor crash.

I'm simply trying to set the visibility on a widget after the return and no matter what I do or which widget I mess with I get this error. If I just simply PrintString() after the invoke, everything is fine. 

I'm pulling my hair out right now trying to solve this. Thanks for any help!

**Comment by nxrighthere** [comment, marked as answer]
Please, provide a blueprint graph with a C# code for reproduction.

**Reply by CrazyguyFPV** [reply]
I solved the problem. I was (hoping) trying to call an async function and awaiting inside the C# code. That was causing problems so I've reconstructed the C# to be sync (works for my case) and everything is working as intended.
